mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -220,10 +221,11 @@ class LinearSVC @Since("2.2.0") (
     instr.logNumFeatures(numFeatures)
 
     if (numInvalid != 0) {
-      val msg = s"Classification labels should be in [0 to ${numClasses - 1}]. " +
-        s"Found $numInvalid invalid labels."
+      val msg = log"Classification labels should be in " +
+        log"${MDC(RANGE_CLASSIFICATION_LABELS, s"[0 to ${numClasses - 1}]")}. " +
```

Comment:
I am thinking about making the log keys generic. How about making it `RANGE` here.

Reply:
Yeah